src/parachain/escrow.ts

```diff
@@ -297,6 +302,11 @@ export class DefaultEscrowAPI implements EscrowAPI {
         return decodeFixedPointType(rawTotalStake);
     }
 
+    async getEscrowTotalStakeAsGovernanceToken(): Promise<MonetaryAmount<Currency<GovernanceUnit>, GovernanceUnit>>  {
+        const rawTotalStake = await this.api.query.escrowRewards.totalStake();
+        return newMonetaryAmount(rawTotalStake.toString(), this.governanceCurrency as Currency<GovernanceUnit>);
```

Comment:
You should be using `getEscrowTotalStake` here, which parses parachain data to Planck. Something like
```suggestion
        return newMonetaryAmount((await this.getEscrowTotalStake()), this.governanceCurrency as Currency<GovernanceUnit>);
```

Reply:
But `getEscrowTotalStake()` will be renamed to `getEscrowTotalStakeRaw`